src/edu/csus/ecs/pc2/core/imports/clics/EventFeed.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 1989-2022 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.imports.clics;
+
+import java.util.HashMap;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.model.JSONObjectMapper;
+
+/**
+ * 
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ */
+
+//@JsonIgnoreProperties(ignoreUnknown = true)
+public class EventFeed {
+
+    // {"type": "teams", "id": "289de609-b1a0-40d8-a9e0-54a30e752e69", "op": "create", "data": {"id": "169347", "name": "Colin Peppler", "icpc_id": null, "group_ids": [], "display_name": "Virginia
+    // Tech", "organization_id": "vt_edu"}}
+
+    @JsonProperty
+    private String type;
+
+    @JsonProperty
+    private String id;
+
+    @JsonProperty
+    private String op;
+
+    @JsonProperty
+    HashMap<String, Object> data;
```

Comment:
I'm not sure what happens if the 'data' member is null (as in a delete event) on the feed.  Will that cause an exception?

Reply:
I have not seen any null data in the event feeds, so unlikely to happen.

Reply:
I think what @johnbrvc meant (JohnB, correct me if I'm wrong) is that whereas in the old EF format we would have seen an "op = delete" field, in the 2022-07 (and forward) EF formats we don't get an "op" -- rather, a "delete" is indicated by the occurrence of a "data:null" field.  So you may not have seen any, because there (probably) aren't any in the particular 2202-07 event feeds we've had so far (just one, from pretest2).   I *think* what JohnB was asking is, what happens in a HashMap if we get a "data:null" in the event feed (which we might) and we try to put that in the HashMap?  (Again, JohnB: speak up if I've misinterpreted the issue...)

Reply:
For now pushed code to ignore any event where data:null rather than spend an unknown amount of
time on handling a delete.
If the delete has an impact on the comparison, that will show up in the comparison/scoreboard like any
other unhandled data.

Reply:
When parsing and finds data:null, the EventFeed.getData() returns null.

Reply:
What @clevengr said is what I was thinking.  The feed we have does not delete anything (that I could find), at least in terms of submissions and judgements, but, that doesn't mean that Kattis doesn't.  But, as long as null is handled, that's all I was concerned about, and it sounds like it is.

Reply:
pushed code earlier to log when data:null